### PR TITLE
fix: fallback to older asciidoctor version (#597)

### PR DESCRIPTION
**Pull request by tkrop**
fixes 597.

I restructured the scripts introducing a make file to factor out the redundant part between Travis and local builds. I also picky backed a fix to the rules generation script I found looking at the log lines, as well as an updated of the maintainer list.

### CODE REVIEW

**Comment by tkrop** [comment]
:+1:

**Comment by tfrauenstein** [comment]
👍

**Comment by tkrop** [comment]
:+1:

**Comment by tkrop** [comment]
:+1:

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,63 @@
+DOCKER := asciidoctor/docker-asciidoctor:1.1.0
+DIRMOUNTS := /documents
+DIRCONTENTS := chapters
+DIRSCRIPTS := scripts
+DIRBUILDS := output
+DIRWORK := $(shell pwd -P)
+
+.PHONY: all clean pull check assets
+.PHOMY: rules html pdf epub force
```
Comment (ePaul) [inline Makefile:9]: What does `.PHOMY` mean?

**Comment by tkrop** [comment]
Hey, I also got an advice from the asciidoctors. how to use the latest version again.

**Comment by tkrop** [comment]
:+1:

**Comment by tfrauenstein** [comment]
👍

**Pull request merged**